crates/db_schema/src/lib.rs

```diff
@@ -115,6 +115,9 @@ pub enum ListingType {
   Local,
   /// Content only from communities you've subscribed to.
   Subscribed,
+  #[serde(rename = "Moderator View")]
```

Comment:
Get rid of this.

Reply:
In https://github.com/LemmyNet/lemmy-js-client/pull/172, I added the listing type as "Moderator View". Since a space in the enum name in the rust code would be impossible / very ugly, I used the rename attribute.

Should the listing type be renamed to "ModeratorView" or just "Moderator"?

Just removing this line will lead to the listing type "Moderator View" not being recognized as ModeratorView.

Reply:
Renaming is more confusing, plus that code is auto-generated and Im not sure it can handle the serde rename correctly.

Reply:
Alright, this will need another change to "ModeratorView" in https://github.com/LemmyNet/lemmy-js-client/blob/main/src/types/ListingType.ts then. Should I open another PR there or will the file automatically be updated when these changes here are merged?

Sorry for the inconvenience, I tried to be clever and update lemmy-js-client first so that the api tests could be rewritten to fit the new implementation without needing to wait for another PR in the middle of development. I didn't give much thought to the space character in the string until trying to make the backend work.

Reply:
Yep, definitely remove this renaming. If necessary I can do an `rc` build of lemmy-js-client. Ping me if / when you need that.

Reply:
@dessalines  If you could change "Moderator View" to "ModeratorView", somewhat reverting the changes in [981c866930de1552338f81d43b5c01431f480759](https://github.com/LemmyNet/lemmy-js-client/pull/172/commits/981c866930de1552338f81d43b5c01431f480759) and create a new official release (candidate), that would be great.